src/globals/_variables.scss

```diff
@@ -27,3 +27,7 @@ $ads-padding-large: 2rem;
 $ads-background-desktop: rgb(217 217 217 / 40%);
 $ads-background-color-black: rgb(0 0 0 / 30%);
 $ads-text-input: #a6a6a6;
+$ads-violetaintenso-main: #7a44ff;
```

Comment:
review the figma the color of tokens names that names needs to match with your names in the the scss globals